RpcSystemAssetTracker/RpcSystemAssetTrackerPlugin.cs

```diff
@@ -156,88 +156,36 @@ private bool ProcessBlock(Snapshot snapshot, Block block)
 
                 if (_shouldTrackHistory)
                 {
-                    UInt160 from = UInt160.Zero;
-                    // need to get all the neo and gas outputs from the inputs
-                    List<TransactionOutput> prevOutputs = new List<TransactionOutput>();
-                    foreach (var group in tx.Inputs.GroupBy(p => p.PrevHash))
+                    // treat each input as a "sent"
+                    for (int i = 0; i < tx.Inputs.Length; i++)
                     {
-                        TransactionState txPrev = transactionsCache[group.Key];
-                        foreach (CoinReference input in group)
+                        var input = tx.Inputs[i];
```

Comment:
foreach?

Reply:
I used `foreach` before, but I need the index as part of the storage key, so `for` is more suitable here.